documentation/book/assembly-kafka-broker-replicas.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-broker-replicas-{context}']
+
+= Replicas
+
+Kafka brokers can run with a different number of nodes.
```

Comment:
I'd replace this whole paragraph with:

Kafka cluster can run with many brokers and Kafka brokers can run with various numbers of nodes. The number of brokers used for the Kafka cluster is defined in the `Kafka` resource. The best number of brokers for your cluster has to be determined based on your specific use case.